pages/contact.js

```diff
@@ -0,0 +1,75 @@
+import Link from 'next/link';
+import Head from 'components/head';
+import Section from 'components/_common_/Section/Section';
+import OutboundLink from 'components/_common_/OutboundLink/OutboundLink';
+import EmailIcon from 'static/images/icons/FontAwesome/envelope-solid.svg';
+import HomeIcon from 'static/images/icons/FontAwesome/home-solid.svg';
+import SlackIcon from 'static/images/icons/FontAwesome/slack-hash.svg';
+import styles from './styles/contact.css';
+
+export default () => (
+  <>
+    <Head title="Contact Us" />
+
+    <Section title="Contact Us">
+      <p className={styles.centerText}>
+        We are a decentralized community of hard-working volunteers, and we love hearing feedback,
+        comments, and suggestions!
+        <br />
+        <br />
+        The best way to reach our staff and our members is by{' '}
+        <Link href="/join">
+          <a>joining Operation Code</a>
+        </Link>{' '}
+        to receive an invite to our team, including our Slack chat. You can also reach us via email,
+        physical mail, or via{' '}
+        <OutboundLink
+          href="//twitter.com/operation_code"
+          analyticsEventLabel="Contact Us - Twitter"
+        >
+          Twitter
+        </OutboundLink>
+      </p>
+    </Section>
+    <Section theme="white" headingLines={false} contentClassName={styles.centerSection}>
+      <div className={styles.flexContainer}>
+        {/* Columns */}
+        <div className={styles.column}>
+          <h5>Email</h5>
+          <EmailIcon className={styles.icon} />
+          <OutboundLink
```

Comment:
For both of the OutboundLink implementations, can you try to implement the Icon + OutboundLink child via the `components/_common_/Badge` component?

After that, you should be able to wrap the whole thing nicely like so:

```html
<OutboundLink
  analyticsEventLabel="Contact Us - Email"
  hasIcon={false}
  href="mailto:staff@operationcode.org"
/>
  <Badge
    label="staff@operationcode.org"
    svgComponent={<EmailIcon className={styles.icon} />}
  />
</OutboundLink>
```

Note: You'll need to change `<Badge>` to expect string OR node prop types to get the mailing address one working nicely.

Reply:
done